database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -18,7 +19,9 @@ public function up()
             $table->string('name');
             $table->string('email')->unique();
             $table->timestamp('email_verified_at')->nullable();
-            $table->string('password');
+            $table->string('password')->nullable(
+                Features::hasSocialiteFeatures()
```

Comment:
what happens if someone enables SocialiteFeatures after the migration did run?
Would if make sense to make this always nullable and check the nullable rule on the saving of a new user?

Reply:
The idea with Jetstream is that it's a scaffold for new app to get up and running easily with auth. Since this feature is optional, you would need to determine whether or not your app/product/service requires socialite feature prior to release. Nots ping so, IMO, would be the improper way to start a project

Reply:
@joelbutcher we would have to note to migrate after setting up config in docs. 2fa is has a separate migration for their table adjustments, maybe be good to follow suit.